107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -50,6 +51,8 @@
 
   private static final URI URI_DEFAULT;
 
+  private static final AtomicInteger CONFIG_COUNT = new AtomicInteger(0);
```

Comment:
@kcleereman Your previous version was better - just make the inner class and added getter package private. Here this atomic is still public and just plain useless but for the test purpose.